topics/admin/tutorials/heterogeneous-compute/slides.html

```diff
@@ -136,3 +136,7 @@
     * [https://github.com/galaxyproject/pulsar](https://github.com/galaxyproject/pulsar)
 * Pulsar Ansible
     * [https://github.com/galaxyproject/ansible-pulsar](https://github.com/galaxyproject/ansible-pulsar)
+
+# Exercise
```

Comment:
Please remove this section, we don't want to link to outdated training resources.

Reply:
ok